uaclient/entitlements/fips.py

```diff
@@ -12,6 +12,74 @@
 event = event_logger.get_event_logger()
 
 
+FIPS_CONDITIONAL_PACKAGES = {
+    "xenial": [
+        "strongswan",
```

Comment:
I think we should have the list of those package inside auxiliary variables and reference then on the dicts

Reply:
Thats a good idea! I'll make this change.

Reply:
Updated!